modules/assim.sequential/R/sda_weights_site.R

```diff
@@ -0,0 +1,47 @@
+#' Calculate ensemble weights for each site at time t.
+#'
+#' @param FORECAST FORECAST object built within the sda.enkf_MultiSite function. 
+#' @param ANALYSIS ANALYSIS object built within the Analysis_sda_multisite function. 
+#' @param t exact number of t inside the sda.enkf_MultiSite function.
+#' @param ens number of ensemble members.
+#'
+#' @return list of weights associated with each ensemble member of each site.
+#' @export
+#'
+#' @examples
+#' @author Dongchen Zhang and Hamzed
```

Comment:
If you include Hamze Dokoohaki as an author (and in the note below) you'll want to spell his name correctly